### PR TITLE
Acm-13465: Remove compliance language

### PR DESCRIPTION
**Pull request by Randy424**
regarding: https://issues.redhat.com/browse/ACM-13465

Do not merge before #3785

### CODE REVIEW

**Comment by KevinFCormier** [comment]
I think someone from GRC should review as they have just updated the strings here recently. @mprahl @yiraeChristineKim can you take a look please? These are values that appear in CSV exported data.

**Comment by yiraeChristineKim** [comment]
@showeimer

File: frontend/src/routes/Governance/policies/Policies.tsx [hunk 1]
```diff
@@ -258,7 +259,10 @@ export default function PoliciesPage() {
         order: 8,
         isDefault: false,
         exportContent: (item) => {
-          return item.source ? item.source : '-'
+          const policyAutomationMatch = policyAutomations.find(
+            (pa: PolicyAutomation) => pa.spec.policyRef === item.policy.metadata.name
+          )
+          return policyAutomationMatch?.metadata.name ? policyAutomationMatch?.metadata.name : '-'
```
Comment (showeimer) [inline frontend/src/routes/Governance/policies/Policies.tsx:265]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing ?

**Review by showeimer — APPROVED** [review]
lgtm

**Review by yiraeChristineKim — APPROVED** [review]

**Comment by Randy424** [comment]
Thank you for reviewing @yiraeChristineKim @showeimer 
I've rebased and adopted Sho's suggestions, would you mind giving a final approval @KevinFCormier?

**Review by mprahl — APPROVED** [review]

**Pull request merged**